File: app/src/main/java/net/sunniwell/aidl/bean/ColumnBean.java
package net.sunniwell.aidl.bean;

import java.io.Serializable;

public class ColumnBean implements Serializable {
    private static final long serialVersionUID = 1;

    /* renamed from: id */
    private int f300id;
    private boolean leaf;
    private int pid;
    private String title;
    private String type;

    public ColumnBean() {
    }

    public ColumnBean(int id, int pid2, String title2, String type2, boolean leaf2) {
        this.f300id = id;
        this.pid = pid2;
        this.title = title2;
        this.type = type2;
        this.leaf = leaf2;
    }

    public int getId() {
        return this.f300id;
    }

    public void setId(int id) {
        this.f300id = id;
    }

    public int getPid() {
        return this.pid;
    }

    public void setPid(int pid2) {
        this.pid = pid2;
    }

    public String getTitle() {
        return this.title;
    }

    public void setTitle(String title2) {
        this.title = title2;
    }

    public String getType() {
        return this.type;
    }

    public void setType(String type2) {
        this.type = type2;
    }

    public boolean isLeaf() {
        return this.leaf;
    }

    public void setLeaf(boolean leaf2) {
        this.leaf = leaf2;
    }

    public boolean equals(Object bean) {
        if (bean == null) {
            return false;
        }
        if (this == bean) {
            return true;
        }
        if (!(bean instanceof ColumnBean)) {
            return false;
        }
        ColumnBean other = (ColumnBean) bean;
        if (this.f300id != other.getId() || this.pid != other.getPid()) {
            return false;
        }
        if (this.title == null) {
            if (other.getTitle() != null) {
                return false;
            }
        } else if (!this.title.equals(other.getTitle())) {
            return false;
        }
        if (this.type == null) {
            if (other.getType() != null) {
                return false;
            }
        } else if (!this.type.equals(other.getType())) {
            return false;
        }
        return true;
    }

    public String toString() {
        return "id = " + this.f300id + " pid = " + this.pid + " title = " + this.title + " type  = " + this.type;
    }
}

File: app/src/main/java/com/eliving/tv/aidl/bean/ColumnBean.kt
package com.eliving.tv.aidl.bean

import java.io.Serializable

class ColumnBean : Serializable {
    /* renamed from: id */
    var id = 0
    var isLeaf = false
    var pid = 0
    var title: String? = null
    var type: String? = null

    constructor() {}
    constructor(
        id: Int,
        pid2: Int,
        title2: String?,
        type2: String?,
        leaf2: Boolean
    ) {
        this.id = id
        pid = pid2
        title = title2
        type = type2
        isLeaf = leaf2
    }

    override fun equals(bean: Any?): Boolean {
        if (bean == null) {
            return false
        }
        if (this === bean) {
            return true
        }
        if (bean !is ColumnBean) {
            return false
        }
        val other = bean
        if (id != other.id || pid != other.pid) {
            return false
        }
        if (title == null) {
            if (other.title != null) {
                return false
            }
        } else if (title != other.title) {
            return false
        }
        if (type == null) {
            if (other.type != null) {
                return false
            }
        } else if (type != other.type) {
            return false
        }
        return true
    }

    override fun toString(): String {
        return "id = " + id + " pid = " + pid + " title = " + title + " type  = " + type
    }

    companion object {
        private const val serialVersionUID: Long = 1
    }
}